README.md

```diff
@@ -14,4 +14,8 @@ You will also see any lint errors in the console.
 
 ## Deploying
 
-TBC
+Automatic deploys are enabled for:
+`develop` to `dtpr-staging-normative`
+`release/production` to `dtpr-main-normative`
```

Comment:
Would either add context that these are heroku apps, or add URLs so context isn't needed

Reply:
ok just added more context that they're heroku apps + the folder name